themes/ddbasic/sass/components/field.scss

```diff
@@ -300,3 +300,49 @@ div.search-field-in-content--message {
     }
   }
 }
+
+// ==========================================================================
+// Ting reference
+// ==========================================================================
+.ting-reference-item {
+  width: 260px;
+  margin-right: 28px;
+  text-align: left;
+
+  // Tablet
+  @include media($tablet) {
+    width: 218px;
+    margin-right: 18px;
+  }
+
+  // Mobile
+  @include media($mobile) {
+    width: 218px;
+    margin-right: 18px;
+  }
+
+  .placeholder {
```

Comment:
[I think I have seen this before](https://github.com/ding2/ding2/blob/af8922fc3ff45e6344e07daa43cdf51b10ed5830/themes/ddbasic/sass/components/ding-carousel.scss#L136) and it seems generally applicable. Consider refactoring into a mixin or the like.

Reply:
That might be the case but I can't see any mixin's in other than the contrib folder in the theme?